maven-java-11/multimodule-maven-project/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.multimodule-maven-project</groupId>
```

Comment:
All of the other modules declare a parent. Do you think this one should declare maven-java-11 as its parent? Just wondering if it was omitted intentionally or not.

Reply:
Yes. I think this one should declare maven-java-11 as its parent. I'll add the tags.